### ISSUE TITLE
Building fails on Windows 8.1

### ISSUE THREAD

**Issue by jgeurts** [atom/electron-starter#79]
Trying to build the project in Windows 8.1 and I'm running into an issue seemingly related to keytar:

```C:\Dev\electron>script\bootstrap
Node: v0.10.33
npm: v1.4.28
Python: v2.7.9
Installing build modules...
gypgyp
npm ERR! runas@1.0.1 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the runas@1.0.1 install script.
npm ERR! This is most likely a problem with the runas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls runas
npm ERR! There is likely additional logging output above.
npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\\Program Files (x86)\\nodejs\\\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\
\npm-cli.js" "--userconfig=C:\\Dev\\electron\\.npmrc" "install" "--quiet"
npm ERR! cwd C:\Dev\electron\build
npm ERR! node -v v0.10.33
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE
npmC:\Dev\electron>script\bootstrap
```

After running: `npm config set msvs_version 2013 --global` and now get the following error when building:

```C:\Dev\electron>script\bootstrap
Node: v0.10.33
npm: v1.4.28
Python: v2.7.9
Installing build modules...
Installing apm...
gypgyp
npm ERR! keytar@2.1.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the keytar@2.1.0 install script.
npm ERR! This is most likely a problem with the keytar package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls keytar
npm ERR! There is likely additional logging output above.
npm ERR! System Windows_NT 6.2.9200
npm ERR! command "node" "C:\\Dev\\electron\\build\\node_modules\\npm\\bin\\npm-cli.js" "--userconfig=C:\\Dev\\electron\\.npmrc" "install" "--quiet"
npm ERR! cwd C:\Dev\electron\apm
npm ERR! node -v v0.10.33
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE
npmC:\Dev\electron>
```

**Comment by paulcbetts** [comment]
You probably have stuff still hanging around from your previous failed build. `git clean -xdf` and remove `%temp%\eightoheight-build` then try again

**Comment by jgeurts** [comment]
Unfortunately, after running the git clean command, I'm seeing the same result. The temp build directory didn't exist. Any other thoughts?

**Comment by jgeurts** [comment]
If it helps, running `script\grunt -v` produces the following:

```C:\Dev\electron-starter>script\grunt -v
Initializing
Command-line options: --gruntfile=C:\Dev\electron-starter\build\Gruntfile.coffee, --verbose

Reading "Gruntfile.coffee" Gruntfile...OK

Registering Gruntfile tasks.

Registering "grunt-bower-task" local Npm module tasks.
Reading C:\Dev\electron-starter\build\node_modules\grunt-bower-task\package.json...OK
Parsing C:\Dev\electron-starter\build\node_modules\grunt-bower-task\package.json...OK
Loading "bower_task.js" tasks...OK
+ bower

Registering "grunt-coffeelint" local Npm module tasks.
Reading C:\Dev\electron-starter\build\node_modules\grunt-coffeelint\package.json...OK
Parsing C:\Dev\electron-starter\build\node_modules\grunt-coffeelint\package.json...OK
Loading "coffeelint.js" tasks...OK
+ coffeelint

Registering "grunt-lesslint" local Npm module tasks.
Reading C:\Dev\electron-starter\build\node_modules\grunt-lesslint\package.json...OK
Parsing C:\Dev\electron-starter\build\node_modules\grunt-lesslint\package.json...OK
Loading "less-lint-task.js" tasks...OK
+ lesslint
Loading "lint-utils.js" tasks...OK
>> No tasks were registered or unregistered.


Registering "grunt-cson" local Npm module tasks.
Reading C:\Dev\electron-starter\build\node_modules\grunt-cson\package.json...OK
Parsing C:\Dev\electron-starter\build\node_modules\grunt-cson\package.json...OK
Loading "cson-task.js" tasks...OK
+ cson

Registering "grunt-contrib-csslint" local Npm module tasks.
Reading C:\Dev\electron-starter\build\node_modules\grunt-contrib-csslint\package.json...OK
Parsing C:\Dev\electron-starter\build\node_modules\grunt-contrib-csslint\package.json...OK
Loading "csslint.js" tasks...OK
+ csslint

Registering "grunt-contrib-coffee" local Npm module tasks.
Reading C:\Dev\electron-starter\build\node_modules\grunt-contrib-coffee\package.json...OK
Parsing C:\Dev\electron-starter\build\node_modules\grunt-contrib-coffee\package.json...OK
Loading "coffee.js" tasks...OK
+ coffee

Registering "grunt-contrib-less" local Npm module tasks.
Reading C:\Dev\electron-starter\build\node_modules\grunt-contrib-less\package.json...OK
Parsing C:\Dev\electron-starter\build\node_modules\grunt-contrib-less\package.json...OK
Loading "less.js" tasks...OK
+ less

Registering "grunt-shell" local Npm module tasks.
Reading C:\Dev\electron-starter\build\node_modules\grunt-shell\package.json...OK
Parsing C:\Dev\electron-starter\build\node_modules\grunt-shell\package.json...OK
Loading "shell.js" tasks...OK
+ shell

Registering "grunt-build-atom-shell" local Npm module tasks.
Reading C:\Dev\electron-starter\build\node_modules\grunt-build-atom-shell\package.json...OK
Parsing C:\Dev\electron-starter\build\node_modules\grunt-build-atom-shell\package.json...OK
Loading "build-atom-shell-task.coffee" tasks...OK
+ build-atom-shell, rebuild-atom-shell, rebuild-native-modules
Loading "task-helpers.coffee" tasks...OK
>> No tasks were registered or unregistered.

Registering "grunt-atom-shell-installer" local Npm module tasks.
Reading C:\Dev\electron-starter\build\node_modules\grunt-atom-shell-installer\package.json...OK
Parsing C:\Dev\electron-starter\build\node_modules\grunt-atom-shell-installer\package.json...OK
Loading "index.js" tasks...OK
+ create-windows-installer

Registering "grunt-peg" local Npm module tasks.
Reading C:\Dev\electron-starter\build\node_modules\grunt-peg\package.json...OK
Parsing C:\Dev\electron-starter\build\node_modules\grunt-peg\package.json...OK
Loading "peg.js" tasks...OK
+ peg

Registering "tasks" tasks.
Loading "build-task.coffee" tasks...OK
+ build
Loading "check-licenses-task.coffee" tasks...OK
+ check-licenses
Loading "clean-task.coffee" tasks...OK
+ clean, partial-clean
Loading "codesign-task.coffee" tasks...OK
+ codesign
Loading "copy-info-plist-task.coffee" tasks...OK
+ copy-info-plist
Loading "dump-symbols-task.coffee" tasks...OK
+ dump-symbols
Loading "generate-asar.coffee" tasks...OK
+ generate-asar
Loading "generate-license-task.coffee" tasks...OK
+ generate-license
Loading "install-task.coffee" tasks...OK
+ install
Loading "license-overrides.coffee" tasks...OK
>> No tasks were registered or unregistered.
Loading "mkdeb-task.coffee" tasks...OK
+ mkdeb
Loading "mkrpm-task.coffee" tasks...OK
+ mkrpm
Loading "output-disk-space.coffee" tasks...OK
+ output-disk-space
Loading "output-long-paths-task.coffee" tasks...OK
+ output-long-paths
Loading "set-exe-icon-task.coffee" tasks...OK
+ set-exe-icon
Loading "set-version-task.coffee" tasks...OK
+ set-version
Loading "spec-task.coffee" tasks...OK
+ run-specs
Loading "task-helpers.coffee" tasks...OK
>> No tasks were registered or unregistered.
Reading package.json...OK
Parsing package.json...OK
Initializing config...OK
Loading "Gruntfile.coffee" tasks...OK
+ ci, compile, default, lint, test

No tasks specified, running default tasks.
Running tasks: default

Running "default" task

Running "build-atom-shell" task
Verifying properties build-atom-shell.buildDir, build-atom-shell.tag, build-atom-shell.projectName, build-atom-shell.pro
ductName exist in config...OK
>> Running: git remote set-url origin https://github.com/atom/atom-shell
>> Running: git fetch origin
>> Running: git reset --hard HEAD
>> Running: git checkout v0.24.0
>> Rigging atom.gyp to have correct name
Reading C:\Users\User\AppData\Local\Temp\eightoheight-build\atom-shell\atom.gyp...OK
Writing C:\Users\User\AppData\Local\Temp\eightoheight-build\atom-shell\atom.gyp...OK
>> bootstrap appears to have been run, skipping it to save time!
>> Running: python script/build.py -c Release -t eightoheight
>>
>> Rigging atom.gyp to have correct name
Reading C:\Users\User\AppData\Local\Temp\eightoheight-build\atom-shell\atom.gyp...OK
Writing C:\Users\User\AppData\Local\Temp\eightoheight-build\atom-shell\atom.gyp...OK
>> Running: python script/bootstrap.py -v
>> Running: python script/build.py -c Release -t eightoheight
>>
Warning: null Use --force to continue.

Aborted due to warnings.
```

**Comment by jgeurts** [comment]
I made sure that I only have x86 versions of node 0.10 and python 2.7... still getting the same error, unfortunately

**Comment by jgeurts** [comment]
The unfortunate part of all of this is that I'm able to npm install keytar without issue...

**Comment by paulcbetts** [comment]
This looks different than the keytar crash, this just looks like you're failing to build Electron. What happens if you go into `C:\Users\User\AppData\Local\Temp\eightoheight-build\atom-shell` and run `python script/build.py -c Release -t eightoheight`

**Comment by marceln** [comment]
I just had the same issue with packages `runas@2.0.0` and `nslog@2.0.0`. I'm also on 8.1 x64 with VS2013 and Node 0.10.38, Python 2.7.9. 

It seems that building those two packages through the build script does not work (didn't investigate further). What I did, after the errors, was to uninstall the local packages and install them globally with npm. That worked and then I ran again grunt, which finally built the sample app. 

For me this `python script/build.py -c Release -t eightoheight` also failed earlier today, but I ended up setting the VC++ build environment manually at the very beginning (it's vcvarsall.bat in C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC)

**Comment by jgeurts** [comment]
@marceln, unfortunately installing the packages globally didn't resolve the keytar issue...

@paulcbetts running the command you mentioned gives the following:

```
ninja: Entering directory `out\R'
[1/81] CXX obj\vendor\native_mate\native_mate\eightoheight_lib.dictionary.obj
[2/81] CXX obj\vendor\native_mate\native_mate\eightoheight_lib.function_template.obj
[3/81] CXX obj\chromium_src\chrome\browser\printing\eightoheight_lib.print_job_worker_owner.obj
[4/81] CXX obj\chromium_src\chrome\browser\printing\eightoheight_lib.print_job_worker.obj
[5/81] CXX obj\chromium_src\chrome\browser\eightoheight_lib.browser_process.obj
[6/81] CXX obj\chromium_src\chrome\browser\printing\eightoheight_lib.print_view_manager_basic.obj
[7/81] CXX obj\chromium_src\chrome\browser\printing\eightoheight_lib.print_view_manager_base.obj
[8/81] CXX obj\chromium_src\chrome\browser\speech\eightoheight_lib.tts_platform.obj
[9/81] CXX obj\chromium_src\chrome\browser\printing\eightoheight_lib.printer_query.obj
[10/81] CXX obj\chromium_src\chrome\common\eightoheight_lib.tts_utterance_request.obj
[11/81] CXX obj\chromium_src\chrome\renderer\printing\eightoheight_lib.print_web_view_helper.obj
[12/81] CXX obj\chromium_src\chrome\browser\speech\eightoheight_lib.tts_win.obj
[13/81] CXX obj\chromium_src\chrome\browser\printing\eightoheight_lib.printing_message_filter.obj
[14/81] CXX obj\atom\renderer\eightoheight_lib.guest_view_container.obj
[15/81] CXX obj\chromium_src\chrome\browser\printing\eightoheight_lib.print_job_manager.obj
FAILED: ninja -t msvc -e environment.x86 -- "C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\bin\amd64_x86\cl.exe" /nologo /showIncludes /FC @obj\chromium_src\chrome\browser\printing\eightoheight_lib.print_view_manager_base.obj.rsp /c ..\..\chromium_src\chrome\browser\printing\print_view_manager_base.cc /Foobj\chromium_src\chrome\browser\printing\eightoheight_lib.print_view_manager_base.obj /Fdobj\eightoheight_lib.cc.pdb 
c:\users\user\appdata\local\temp\eightoheight-build\atom-shell\vendor\brightray\vendor\download\libchromiumcontent\src\base\tuple.h(99) : fatal error C1001: An internal error has occurred in the compiler.
(compiler file 'msc1.cpp', line 1325)
 To work around this problem, try simplifying or changing the program near the locations listed above.
Please choose the Technical Support command on the Visual C++ 
 Help menu, or open the Technical Support help file for more information
        c:\users\user\appdata\local\temp\eightoheight-build\atom-shell\chromium_src\chrome\common\print_messages.h(208) : see reference to class template instantiation 'Tuple<int,int>' being compiled
        c:\users\user\appdata\local\temp\eightoheight-build\atom-shell\chromium_src\chrome\browser\printing\print_view_manager_base.cc(185) : see reference to function template instantiation 'bool PrintHostMsg_DidGetPrintedPagesCount::Dispatch<printing::PrintViewManagerBase,printing::PrintViewManagerBase,void,void(__thiscall printing::PrintViewManagerBase::* )(int,int)>(const IPC::Message *,T *,S *,P *,Method)' being compiled
        with
        [
            T=printing::PrintViewManagerBase
,            S=printing::PrintViewManagerBase
,            P=void
,            Method=void (__thiscall printing::PrintViewManagerBase::* )(int,int)
        ]
Internal Compiler Error in C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\bin\amd64_x86\cl.exe.  You will be prompted to send an error report to Microsoft later.
FAILED: ninja -t msvc -e environment.x86 -- "C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\bin\amd64_x86\cl.exe" /nologo /showIncludes /FC @obj\chromium_src\chrome\renderer\printing\eightoheight_lib.print_web_view_helper.obj.rsp /c ..\..\chromium_src\chrome\renderer\printing\print_web_view_helper.cc /Foobj\chromium_src\chrome\renderer\printing\eightoheight_lib.print_web_view_helper.obj /Fdobj\eightoheight_lib.cc.pdb 
c:\users\user\appdata\local\temp\eightoheight-build\atom-shell\vendor\brightray\vendor\download\libchromiumcontent\src\base\tuple.h(99) : fatal error C1001: An internal error has occurred in the compiler.
(compiler file 'msc1.cpp', line 1325)
 To work around this problem, try simplifying or changing the program near the locations listed above.
Please choose the Technical Support command on the Visual C++ 
 Help menu, or open the Technical Support help file for more information
        c:\users\user\appdata\local\temp\eightoheight-build\atom-shell\chromium_src\chrome\common\print_messages.h(188) : see reference to class template instantiation 'Tuple<bool,bool>' being compiled
        c:\users\user\appdata\local\temp\eightoheight-build\atom-shell\chromium_src\chrome\renderer\printing\print_web_view_helper.cc(648) : see reference to function template instantiation 'bool PrintMsg_PrintPages::Dispatch<printing::PrintWebViewHelper,printing::PrintWebViewHelper,void,void(__thiscall printing::PrintWebViewHelper::* )(bool,bool)>(const IPC::Message *,T *,S *,P *,Method)' being compiled
        with
        [
            T=printing::PrintWebViewHelper
,            S=printing::PrintWebViewHelper
,            P=void
,            Method=void (__thiscall printing::PrintWebViewHelper::* )(bool,bool)
        ]
Internal Compiler Error in C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\bin\amd64_x86\cl.exe.  You will be prompted to send an error report to Microsoft later.
FAILED: ninja -t msvc -e environment.x86 -- "C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\bin\amd64_x86\cl.exe" /nologo /showIncludes /FC @obj\chromium_src\chrome\browser\printing\eightoheight_lib.printing_message_filter.obj.rsp /c ..\..\chromium_src\chrome\browser\printing\printing_message_filter.cc /Foobj\chromium_src\chrome\browser\printing\eightoheight_lib.printing_message_filter.obj /Fdobj\eightoheight_lib.cc.pdb 
c:\users\user\appdata\local\temp\eightoheight-build\atom-shell\vendor\brightray\vendor\download\libchromiumcontent\src\base\tuple.h(99) : fatal error C1001: An internal error has occurred in the compiler.
(compiler file 'msc1.cpp', line 1325)
 To work around this problem, try simplifying or changing the program near the locations listed above.
Please choose the Technical Support command on the Visual C++ 
 Help menu, or open the Technical Support help file for more information
        c:\users\user\appdata\local\temp\eightoheight-build\atom-shell\chromium_src\chrome\common\print_messages.h(201) : see reference to class template instantiation 'Tuple<base::SharedMemoryHandle>' being compiled
        c:\users\user\appdata\local\temp\eightoheight-build\atom-shell\chromium_src\chrome\browser\printing\printing_message_filter.cc(120) : see reference to function template instantiation 'bool PrintHostMsg_DuplicateSection::Dispatch<printing::PrintingMessageFilter,printing::PrintingMessageFilter,void,void(__thiscall printing::PrintingMessageFilter::* )(base::SharedMemoryHandle,base::SharedMemoryHandle *)>(const IPC::Message *,T *,S *,P *,Method)' being compiled
        with
        [
            T=printing::PrintingMessageFilter
,            S=printing::PrintingMessageFilter
,            P=void
,            Method=void (__thiscall printing::PrintingMessageFilter::* )(base::SharedMemoryHandle,base::SharedMemoryHandle *)
        ]
Internal Compiler Error in C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\bin\amd64_x86\cl.exe.  You will be prompted to send an error report to Microsoft later.
ninja: build stopped: subcommand failed.
```

**Comment by marceln** [comment]
@jgeurts: It might be your VS version. Do you run VS2013 update 4? According to this (http://www.chromium.org/developers/how-tos/build-instructions-windows#TOC-System-level-toolchain) no other version is supported.

**Comment by jgeurts** [comment]
@marceln, Thank you, that fixed the issues running `script\grunt`

Unfortunately, running `script\bootstrap` still result in the keytar issue and I would love to figure out why that build is failing...

**Comment by marceln** [comment]
@jgeurts: Not sure. Yesterday I built it on two machines. On one I had errors, but that's because I started off with the wrong node version. On the second one, with a fresh electron-starter checkout it worked fine, no errors. You might want to try a fresh build, by deleting all node packages from AppData.

**Comment by jgeurts** [comment]
I got things working! 

**tl;dr** is that I had to run `npm install` manually in the apm sub-directory.


To provide a bit more information, my configuration is:

* Visual Studio 2013 Professional with update 4 (thanks @marceln)
* Python 2.7.9 x86
* node.js 0.10.38 x86


I deleted the electron-starter directory, cloned a new copy from github. I deleted everything in:

* C:\Users\User\.node-gyp
* C:\Users\User\.config
* C:\Users\User\AppData\local\Temp
* C:\Users\User\AppData\Roaming\npm
* C:\Users\User\AppData\Roaming\npm-cache
* C:\Users\User\AppData\Roaming\bower

I then ran: `npm config set msvs_version 2013 --global`

I restarted the VM just to make sure. Then I ran:
```
C:\>cd dev\electron-starter
C:\dev\electron-starter>script\bootstrap
```

With the following output:

```
C:\Dev\electron-starter>script\bootstrap
Node: v0.10.38
npm: v1.4.28
Python: v2.7.9
Installing build modules...
Installing apm...
gypgyp
npm ERR! keytar@2.1.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the keytar@2.1.0 install script.
npm ERR! This is most likely a problem with the keytar package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls keytar
npm ERR! There is likely additional logging output above.
npm ERR! System Windows_NT 6.2.9200
npm ERR! command "node" "C:\\Dev\\electron-starter\\build\\node_modules\\npm\\bin\\npm-cli.js" "--userconfig=C:\\Dev\\el
ectron-starter\\.npmrc" "install" "--quiet"
npm ERR! cwd C:\Dev\electron-starter\apm
npm ERR! node -v v0.10.38
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE
npmC:\Dev\electron-starter>
```

I then did the following:
```
C:\Dev\electron-starter\npm>npm install
```

and then re-ran the `script\bootstrap` command with success!

**Comment by marceln** [comment]
@jgeurts: Great! Do you have multiple VS versions installed on that machine? Like VS2012, etc? I'm curious why you had tell npm explicitly which version to use. That might very well be a required step when building and two or more VS versions are available.

**Comment by jgeurts** [comment]
@marceln Looks like for whatever reason, Visual Studio 2010 prerequisites are installed on this particular machine. Without explicitly specifying 2013 to npm, I get the following:

```
C:\Dev\electron-starter>npm install runas@2.0.0 --global
|
> runas@2.0.0 install C:\Users\User\AppData\Roaming\npm\node_modules\runas
> node-gyp rebuild


C:\Users\User\AppData\Roaming\npm\node_modules\runas>node "C:\Program Files (x86)\nodejs\node_modules\npm\bin\node-gyp-b
in\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V120\Microsoft.Cpp.Platform.targets(64,5): error MSB8020: The build t
ools for Visual Studio 2010 (Platform Toolset = 'v100') cannot be found. To build using the v100 build tools, please in
stall Visual Studio 2010 build tools.  Alternatively, you may upgrade to the current Visual Studio tools by selecting t
he Project menu or right-click the solution, and then selecting "Upgrade Solution...". [C:\Users\User\AppData\Roaming\n
pm\node_modules\runas\build\runas.vcxproj]
gyp ERR! build error
gyp ERR! stack Error: `msbuild` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files (x86)\nodejs\node_modules\npm\node_modules\node-gyp\lib\buil
d.js:267:23)
gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:820:12)
gyp ERR! System Windows_NT 6.2.9200
gyp ERR! command "node" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "
rebuild"
gyp ERR! cwd C:\Users\User\AppData\Roaming\npm\node_modules\runas
gyp ERR! node -v v0.10.38
gyp ERR! node-gyp -v v1.0.1
gyp ERR! not ok

npm ERR! runas@2.0.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the runas@2.0.0 install script.
npm ERR! This is most likely a problem with the runas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls runas
npm ERR! There is likely additional logging output above.
npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\\Program Files (x86)\\nodejs\\\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\
\npm-cli.js" "install" "runas@2.0.0" "--global"
npm ERR! cwd C:\Dev\electron-starter
npm ERR! node -v v0.10.38
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE
npm ERR! not ok code 0
```

Maybe that is the source of all of this pain...

**Comment by jgeurts** [comment]
I gave up on the Windows 8.1 VM and started from scratch with a Windows 10 VM... everything built flawlessly on that...
